Generator/DoctrineCrudGenerator.php

```diff
@@ -296,4 +293,14 @@ protected function getRecordActions()
             return in_array($item, array('show', 'edit'));
         });
     }
+
+    public static function getRouteNamePrefix($prefix)
+    {
+        $prefix = preg_replace('/{(.*?)}/', '', $prefix);   // {foo}_bar -> _bar
+        $prefix = str_replace('/', '_', $prefix);
+        $prefix = preg_replace('/(_)\\1+/', '$1', $prefix); // foo__bar -> foo_bar
```

Comment:
should be simplified with `preg_replace('/_+/', '_', $prefix)`... or am I missing something?